<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>900</width>
    <height>661</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Expanding" vsizetype="Maximum">
    <horstretch>255</horstretch>
    <verstretch>255</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>900</width>
    <height>661</height>
   </size>
  </property>
  <property name="sizeIncrement">
   <size>
    <width>100</width>
    <height>100</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Convertor</string>
  </property>
  <property name="windowIcon">
   <iconset resource="background.qrc">
    <normaloff>:/back/happyCat.jpg</normaloff>:/back/happyCat.jpg</iconset>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <widget class="QWidget" name="centralwidget">
   <property name="minimumSize">
    <size>
     <width>725</width>
     <height>385</height>
    </size>
   </property>
   <widget class="QLabel" name="SelectInitBase">
    <property name="geometry">
     <rect>
      <x>125</x>
      <y>180</y>
      <width>161</width>
      <height>30</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Cambria</family>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="layoutDirection">
     <enum>Qt::RightToLeft</enum>
    </property>
    <property name="text">
     <string>Select initial base:</string>
    </property>
    <property name="textFormat">
     <enum>Qt::AutoText</enum>
    </property>
    <property name="scaledContents">
     <bool>false</bool>
    </property>
    <property name="wordWrap">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QRadioButton" name="sourceBinary">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>220</y>
      <width>50</width>
      <height>20</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Cambria</family>
     </font>
    </property>
    <property name="layoutDirection">
     <enum>Qt::LeftToRight</enum>
    </property>
    <property name="text">
     <string>2</string>
    </property>
    <attribute name="buttonGroup">
     <string notr="true">sourceBase</string>
    </attribute>
   </widget>
   <widget class="QRadioButton" name="sourceHexadecimal">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>280</y>
      <width>50</width>
      <height>20</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Cambria</family>
     </font>
    </property>
    <property name="layoutDirection">
     <enum>Qt::LeftToRight</enum>
    </property>
    <property name="text">
     <string>16</string>
    </property>
    <attribute name="buttonGroup">
     <string notr="true">sourceBase</string>
    </attribute>
   </widget>
   <widget class="QRadioButton" name="sourceDecimal">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>250</y>
      <width>51</width>
      <height>20</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>51</width>
      <height>0</height>
     </size>
    </property>
    <property name="font">
     <font>
      <family>Cambria</family>
     </font>
    </property>
    <property name="layoutDirection">
     <enum>Qt::LeftToRight</enum>
    </property>
    <property name="text">
     <string>10</string>
    </property>
    <property name="checked">
     <bool>true</bool>
    </property>
    <attribute name="buttonGroup">
     <string notr="true">sourceBase</string>
    </attribute>
   </widget>
   <widget class="QLabel" name="EnterText">
    <property name="geometry">
     <rect>
      <x>115</x>
      <y>130</y>
      <width>131</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Cambria</family>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="layoutDirection">
     <enum>Qt::RightToLeft</enum>
    </property>
    <property name="text">
     <string>Enter number:</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="EnterLine">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>250</x>
      <y>130</y>
      <width>141</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Cambria</family>
     </font>
    </property>
    <property name="whatsThis">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;You need to enter a number here&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="maxLength">
     <number>32</number>
    </property>
   </widget>
   <widget class="QLabel" name="ResultText">
    <property name="geometry">
     <rect>
      <x>125</x>
      <y>390</y>
      <width>61</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Cambria</family>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Result:</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="newBinary">
    <property name="geometry">
     <rect>
      <x>400</x>
      <y>220</y>
      <width>50</width>
      <height>20</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Cambria</family>
     </font>
    </property>
    <property name="toolTipDuration">
     <number>0</number>
    </property>
    <property name="text">
     <string>2</string>
    </property>
    <property name="checked">
     <bool>true</bool>
    </property>
    <attribute name="buttonGroup">
     <string notr="true">newBase</string>
    </attribute>
   </widget>
   <widget class="QLabel" name="SelectNewBase">
    <property name="geometry">
     <rect>
      <x>365</x>
      <y>180</y>
      <width>140</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Cambria</family>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Select new base:</string>
    </property>
    <property name="textFormat">
     <enum>Qt::AutoText</enum>
    </property>
    <property name="scaledContents">
     <bool>false</bool>
    </property>
    <property name="wordWrap">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QLabel" name="ResultLabel">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>390</y>
      <width>461</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Cambria</family>
      <pointsize>12</pointsize>
      <italic>true</italic>
     </font>
    </property>
    <property name="layoutDirection">
     <enum>Qt::LeftToRight</enum>
    </property>
    <property name="styleSheet">
     <string notr="true">color:rgb(96, 155, 58)
</string>
    </property>
    <property name="text">
     <string>result number</string>
    </property>
   </widget>
   <widget class="QPushButton" name="ConvertButton">
    <property name="geometry">
     <rect>
      <x>235</x>
      <y>343</y>
      <width>120</width>
      <height>40</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Cambria</family>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Convert</string>
    </property>
    <property name="checkable">
     <bool>false</bool>
    </property>
    <property name="autoRepeat">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QRadioButton" name="newDecimal">
    <property name="geometry">
     <rect>
      <x>400</x>
      <y>250</y>
      <width>51</width>
      <height>20</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>51</width>
      <height>0</height>
     </size>
    </property>
    <property name="font">
     <font>
      <family>Cambria</family>
     </font>
    </property>
    <property name="toolTipDuration">
     <number>1</number>
    </property>
    <property name="text">
     <string>10</string>
    </property>
    <attribute name="buttonGroup">
     <string notr="true">newBase</string>
    </attribute>
   </widget>
   <widget class="QRadioButton" name="newHexadecimal">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>400</x>
      <y>280</y>
      <width>50</width>
      <height>20</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Cambria</family>
     </font>
    </property>
    <property name="text">
     <string>16</string>
    </property>
    <attribute name="buttonGroup">
     <string notr="true">newBase</string>
    </attribute>
   </widget>
   <widget class="QLabel" name="ErrorLabel">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>385</x>
      <y>310</y>
      <width>281</width>
      <height>101</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Cambria</family>
      <pointsize>10</pointsize>
      <weight>75</weight>
      <bold>true</bold>
      <kerning>true</kerning>
     </font>
    </property>
    <property name="autoFillBackground">
     <bool>false</bool>
    </property>
    <property name="styleSheet">
     <string notr="true">color:rgb(164, 56, 75)</string>
    </property>
    <property name="text">
     <string>Error</string>
    </property>
    <property name="textFormat">
     <enum>Qt::AutoText</enum>
    </property>
    <property name="scaledContents">
     <bool>false</bool>
    </property>
    <property name="wordWrap">
     <bool>true</bool>
    </property>
    <property name="openExternalLinks">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="SwapButton">
    <property name="geometry">
     <rect>
      <x>275</x>
      <y>250</y>
      <width>60</width>
      <height>30</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Cambria</family>
     </font>
    </property>
    <property name="text">
     <string>&lt;--&gt;</string>
    </property>
   </widget>
   <widget class="QPushButton" name="CopyButton">
    <property name="geometry">
     <rect>
      <x>185</x>
      <y>435</y>
      <width>120</width>
      <height>40</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
      <horstretch>192</horstretch>
      <verstretch>117</verstretch>
     </sizepolicy>
    </property>
    <property name="sizeIncrement">
     <size>
      <width>1920</width>
      <height>1200</height>
     </size>
    </property>
    <property name="font">
     <font>
      <family>Cambria</family>
      <pointsize>7</pointsize>
      <italic>true</italic>
     </font>
    </property>
    <property name="layoutDirection">
     <enum>Qt::LeftToRight</enum>
    </property>
    <property name="autoFillBackground">
     <bool>true</bool>
    </property>
    <property name="text">
     <string>copy to clipboard</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="sourceOther">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>310</y>
      <width>81</width>
      <height>20</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Cambria</family>
     </font>
    </property>
    <property name="layoutDirection">
     <enum>Qt::LeftToRight</enum>
    </property>
    <property name="text">
     <string>Other:</string>
    </property>
    <attribute name="buttonGroup">
     <string notr="true">sourceBase</string>
    </attribute>
   </widget>
   <widget class="QRadioButton" name="newOther">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>400</x>
      <y>310</y>
      <width>81</width>
      <height>20</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Cambria</family>
     </font>
    </property>
    <property name="text">
     <string>Other:</string>
    </property>
    <attribute name="buttonGroup">
     <string notr="true">newBase</string>
    </attribute>
   </widget>
   <widget class="QLineEdit" name="linesourceOther">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>255</x>
      <y>305</y>
      <width>31</width>
      <height>25</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Cambria</family>
     </font>
    </property>
    <property name="layoutDirection">
     <enum>Qt::LeftToRight</enum>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLineEdit" name="linenewOther">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>485</x>
      <y>310</y>
      <width>31</width>
      <height>25</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Cambria</family>
     </font>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QGraphicsView" name="Background">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>-10</y>
      <width>1920</width>
      <height>1200</height>
     </rect>
    </property>
    <property name="autoFillBackground">
     <bool>false</bool>
    </property>
    <property name="styleSheet">
     <string notr="true">background-image: url(:/back/back.png);</string>
    </property>
   </widget>
   <widget class="QLabel" name="Secret">
    <property name="geometry">
     <rect>
      <x>840</x>
      <y>650</y>
      <width>171</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>простите у меня лапки</string>
    </property>
   </widget>
   <widget class="QLabel" name="Cat">
    <property name="geometry">
     <rect>
      <x>480</x>
      <y>80</y>
      <width>300</width>
      <height>300</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">image: url(:/back/waitingCat.jpg)</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="heart">
    <property name="geometry">
     <rect>
      <x>490</x>
      <y>160</y>
      <width>68</width>
      <height>64</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>64</width>
      <height>64</height>
     </size>
    </property>
    <property name="styleSheet">
     <string notr="true">image: url(:/back/heart.png);</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="heart_2">
    <property name="geometry">
     <rect>
      <x>980</x>
      <y>640</y>
      <width>68</width>
      <height>64</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>64</width>
      <height>64</height>
     </size>
    </property>
    <property name="styleSheet">
     <string notr="true">image: url(:/back/heart.png);</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="heart_3">
    <property name="geometry">
     <rect>
      <x>830</x>
      <y>620</y>
      <width>68</width>
      <height>64</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>64</width>
      <height>64</height>
     </size>
    </property>
    <property name="styleSheet">
     <string notr="true">image: url(:/back/heart.png);</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="heart_4">
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>90</y>
      <width>68</width>
      <height>64</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>64</width>
      <height>64</height>
     </size>
    </property>
    <property name="styleSheet">
     <string notr="true">image: url(:/back/heart.png);</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="ster">
    <property name="geometry">
     <rect>
      <x>290</x>
      <y>380</y>
      <width>128</width>
      <height>128</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">image: url(:/back/star.png);</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <zorder>Background</zorder>
   <zorder>heart_4</zorder>
   <zorder>heart_2</zorder>
   <zorder>SelectInitBase</zorder>
   <zorder>sourceBinary</zorder>
   <zorder>sourceHexadecimal</zorder>
   <zorder>sourceDecimal</zorder>
   <zorder>EnterText</zorder>
   <zorder>EnterLine</zorder>
   <zorder>ResultText</zorder>
   <zorder>newBinary</zorder>
   <zorder>SelectNewBase</zorder>
   <zorder>ResultLabel</zorder>
   <zorder>ConvertButton</zorder>
   <zorder>newDecimal</zorder>
   <zorder>newHexadecimal</zorder>
   <zorder>ErrorLabel</zorder>
   <zorder>SwapButton</zorder>
   <zorder>CopyButton</zorder>
   <zorder>sourceOther</zorder>
   <zorder>newOther</zorder>
   <zorder>linesourceOther</zorder>
   <zorder>linenewOther</zorder>
   <zorder>Secret</zorder>
   <zorder>Cat</zorder>
   <zorder>heart</zorder>
   <zorder>heart_3</zorder>
   <zorder>ster</zorder>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>900</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar">
   <property name="sizePolicy">
    <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
     <horstretch>192</horstretch>
     <verstretch>120</verstretch>
    </sizepolicy>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
  </widget>
  <widget class="QToolBar" name="toolBar">
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
 </widget>
 <resources>
  <include location="background.qrc"/>
 </resources>
 <connections/>
 <buttongroups>
  <buttongroup name="newBase"/>
  <buttongroup name="sourceBase">
   <property name="myID" stdset="0">
    <number>0</number>
   </property>
  </buttongroup>
 </buttongroups>
</ui>
